entities/src/secondary_user.rs

```diff
@@ -11,6 +11,7 @@ pub struct Model {
     pub id: u64,
     pub user_id: u64,
     pub description: Option<String>,
+    pub rpc_key: u64,
```

Comment:
lets keep the name consistent. we use "rpc_key" and "rpc_key_id" and "rpc_secret_key_id". i like "secret" in it because it feels safest